store/modules/playback/actions.js

```diff
@@ -173,13 +173,19 @@ export default {
   },
   async validate_barcode({ commit, state, dispatch }, { type, new_value }) {
     const result = TextValidation_plate_barcode.validate(new_value, type, this.state.settings.beta_2_mode);
-    const is_valid = result == "";
+    let is_valid = result == "";
 
     // stop all running processes if either barcode changes regardless of validity
     if (this.state.stimulation.stim_play_state) {
       await this.dispatch("stimulation/stop_stimulation");
       commit("set_barcode_warning", true);
     }
+    if (
```

Comment:
This check should happen in this function https://github.com/CuriBio/mantarray-frontend-components/blob/main/js_utils/text_validation.js#L175